### ISSUE TITLE
Supported versions?

### ISSUE THREAD

**Issue by arichtig** [s12v/newrelic-elasticsearch#72]
We recently updated to version 6.3 of ElasticSearch and are having issues getting it to report to New Relic.  The ports appear to be listening but New Relic is not getting the data from the plugin or even recognizing them as nodes for an instance.  Should this version work correctly on the new version of Elastic Search or is there an update in the works for the new Elastic Search?

**Comment by lauracw100** [comment]
We also upgraded from 6.2.4 to 6.4.2 and the plugins are no longer reporting to New Relic. Any plans to support this plugin going forward to newer versions of Elastic Search?

**Comment by alexlance** [comment]
We tried to use the plugin on a cluster running ES 6.4.3 but there is a crash in the logs:

    SEVERE: An error has occurred
    java.lang.NullPointerException
    at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportThreadPool(NodesStatsReporter.java:191)                                                                                                                                                                      
    at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodeStats(NodesStatsReporter.java:87)
    at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodesStats(NodesStatsReporter.java:24)
    at me.snov.newrelic.elasticsearch.ElasticsearchAgent.pollCycle(ElasticsearchAgent.java:44)
    at com.newrelic.metrics.publish.Runner$PollAgentsRunnable.run(Runner.java:209)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)

I guess the format of the json endpoints has changed perhaps?

**Comment by ghost** [comment]
Same issues here. Getting the above error running ElasticSearch 6.6.0 on Debian 8.
```
java.lang.NullPointerException
        at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportThreadPool(NodesStatsReporter.java:191)
        at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodeStats(NodesStatsReporter.java:87)
        at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodesStats(NodesStatsReporter.java:24)
        at me.snov.newrelic.elasticsearch.ElasticsearchAgent.pollCycle(ElasticsearchAgent.java:44)
        at com.newrelic.metrics.publish.Runner$PollAgentsRunnable.run(Runner.java:209)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by s12v** [comment]
I published a new version 2.4.0 with ES6 support
